src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -354,4 +355,25 @@ public Auth exchangeRefreshToken(String refreshToken) throws SmartcarException {
 
     return this.call(requestBody);
   }
+
+  /**
+   * Determine if a vehicle is compatible with Smartcar.
+   *
+   * @param vin the vin of the vehicle
+   *
+   * @return true if the vehicle is compatible
+   *
+   * @throws SmartcarException when the request is unsuccessful
+   */
+  public boolean compatibility(String vin) throws SmartcarException {
+	Request request = new Request.Builder()
+        .url(this.urlApi + "/compatibility?vin=" + vin)
```

Comment:
We can also use the [`HttpUrl.Builder`](https://square.github.io/okhttp/3.x/okhttp/okhttp3/HttpUrl.html) like we use above for `getAuthUrl`.

```java
HttpUrl url = HttpUrl.parse(this.urlApi).newBuilder()
    .addPathSegment("compatibility")
    .addQueryParameter("vin", vin)
    .build();

Request request = new Request.Builder()
    .url(url)
    ...
```